detection-rules/link_coinbase_phishing_suspicious_link.yml

```
@@ -0,0 +1,17 @@
+name: "Brand impersonation: Coinbase with suspicious link"
+description: "This rule detects Coinbase phishing emails with suspicious links"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and sender.email.domain.root_domain != "coinbase.com"
+  and length(body.links) ==1 
+  and (
+      regex.icontains(coalesce(sender.display_name, subject.subject),
+        '\b[CⒸⓒ𝑪𝒞𝓬𝔠𝕔𝕥𝖈𝖔𝖙𝗌𝗰𝘊𝘤𝙤𝚌𝝰][oₒⓞ𝑜𝒐𝓸𝕠𝖔𝗼𝘐𝙤𝚘𝞸][i𝐼𝑖𝒊𝓲𝕚𝖎𝗜𝘪𝙄𝚒][nⓃ𝑛𝒏𝓷𝔫𝕟𝖓𝙣𝚗]\s?[BⒷⓑ𝑩𝒃𝓫𝔟𝕓𝕥𝗕][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪][s𝑆𝒔𝓢𝔰𝕊𝕊𝖘𝖘𝗦𝘴𝙎𝚂][e𝑒𝒆𝓔𝔢𝕖𝕖𝖊𝖊𝗲𝙀𝚎𝝴𝞮]\b')
+  )
+  and all(body.links,
+  .href_url.domain.root_domain in $free_subdomain_hosts or .href_url.domain.domain in $free_file_hosts)
```

```suggestion
  and all(body.links,
      .href_url.domain.root_domain in $free_subdomain_hosts
      or .href_url.domain.domain in $free_file_hosts
  )
```

line 14 should be indented since it's inside the `all` parens